src/components/dropdown-menu/dropdown-menu.module.scss

```diff
@@ -0,0 +1,39 @@
+@import '../../styles/common/variables.module.scss';
+
+.content {
+    background-color: white;
+    border-radius: 6px;
+    padding: 8px;
+    width: calc(100% + 20px);
+    border: 1px solid $grass1;
+    box-shadow: 0px 12px 60px 0px #0000000e;
+    box-shadow: 0px 12px 32px -16px #0000301b;
```

Comment:
Only the last declaration applied. To apply multiple shadows you need to specify a comma-separated list `box-shadow: shadow1, shadow2`;